browser/resources/settings/brave_overrides/import_data_dialog.ts

```diff
@@ -3,14 +3,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck TODO(petemill): Define types and remove ts-nocheck
-
 import {html, RegisterPolymerTemplateModifications} from 'chrome://resources/brave/polymer_overriding.js'
 import {loadTimeData} from 'chrome://resources/js/load_time_data.js'
 
 RegisterPolymerTemplateModifications({
   'settings-import-data-dialog': (templateContent) => {
-    let checkBoxesParent = templateContent.querySelector('#browserSelect').parentElement;
+    let checkBoxesParent =
+      templateContent.querySelector('#browserSelect').parentElement
```

Comment:
nit:

```suggestion
    const checkBoxesParent =
      templateContent.querySelector('#browserSelect').parentElement
```

also, I'm surprised this doesn't need to be
```suggestion
    const checkBoxesParent =
      templateContent.querySelector('#browserSelect')!.parentElement
```

Reply:
Fixed here 792a25240aedb81bc9901f118c03781fdb5da593. I added the `!` just to be safe, but yeah strangely enough the compiler wasn't complaining about that...